app/language/map.py

```diff
@@ -56,49 +61,54 @@ def get_map():
 def build_map():
 	# re-initialize map
 	map = {}
+	script_dir = os.path.dirname(__file__)
```

Comment:
Yes, this works locally. No clue how it will behave on heroku. Any help
would be appreciated!

Bill Cromie
t:917.215.3576

sent on the move

On Jan 20, 2015 10:53 PM, "Alex" <notifications@github.com> wrote:

> In app/language/map.py
> <https://github.com/aberke/check-list/pull/5#discussion_r23276852>:
>
> > @@ -56,49 +61,54 @@ def get_map():
> >  def build_map():
> >  	# re-initialize map
> >  	map = {}
> > +	script_dir = os.path.dirname(__file__)
>
> A little worried about how the file paths will behave with heroku
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aberke/check-list/pull/5/files#r23276852>.
>